libsupport/include/katana/ErrorCode.h

```diff
@@ -105,6 +106,8 @@ class KATANA_EXPORT ErrorCodeCategory : public std::error_category {
       return "assertion failed";
     case ErrorCode::GraphUpdateFailed:
       return "graph update failed";
+    case ErrorCode::NotBuiltWith:
+      return "the software is not built with this feature";
```

Comment:
Maybe "Katana is not build with this feature" in case this message is displayed in a context where there are multiple packages in use.

Reply:
Changed.

Reply:
bike shed moment: `ErrorCode::FeatureNotEnabled`?

Reply:
I actually agree here: `ErrorCode::FeatureNotEnabled` is definitely better.

Reply:
Changed.